### DISCUSSION TITLE
en/quick/install

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#94] [General]
# en/quick/install

Quick start guide for the GoFrame framework. GoFrame is a modular, low coupling design development framework that includes common basic components and development tools. It's suitable for complete business project frameworks and independent component libraries. The content covers downloading and installing GoFrame, running basic operations, and introducing how to develop simple Web Server API applications.

https://goframe.org/en/quick/install

**Comment by oldme-git** [comment]
awesome
